doc/WebSite/Dependency-Injection.md

```diff
@@ -313,7 +313,7 @@ should add it in pre-initialize method of your module.
 
 You may want to register a specific class that does not fit into the
 conventional registration rules. ASP.NET Boilerplate provides the
-**ITransientDependency** and the **ISingletonDependency** interfaces as a
+**ITransientDependency**, the **ISingletonDependency** and the **IPerWebRequestDependency** interfaces as a
```

Comment:
Order by increasing frequency of reuse? i.e. `ISingletonDependency` last.

Reply:
No special ordering :) You mean this?

ITransientDependency, IPerWebRequestDependency, ISingletonDependency,

Is it important?

Reply:
> You mean this?

Yes.

> Is it important?

No, but makes sense to be ordered.

Reply:
Done. Thanks for suggestions as always :)